docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -97,164 +83,204 @@ can be seen in the Information tab, as shown in the image below. For more inform
     :align: center
 
-    General Configuration for PWM, Red Dots on Important Parameters
+    Communication Parameter
 
-The list below provides a description of the important parameters and details on what they should be set to. **Most paramaters are the same in PWM and DSHOT setups, but ones that differ have
-sections for both setups**:
+.. warning:: **Known Bug:** In version 1.2.5 and earlier versions of the Control Center, there is a bug in the dropdown menu for the Communication parameter. The numbering for options besides Autodetect was improperly configured.
+    If you select a protocol from the list other than Autodetect, the module will actually be listening for the next protocol up in the list instead of what you selected. So in order to listen for PWM, you should select OneShot125.
+    This is addressed in version 1.2.6 and later of the Control Center.
 
-* **Communication**: This parameter controls what type of Hobby Communication protocol the motor will listen for. The motor is capable of listening to a wide variety of hobby protocols,
-  which can be selected from the drop-down box. Once a motor hears a command in its selected hobby protocol, it will only listen for that type of command until power cycled, so you will
-  not be able to connect with Control Center until you power cycle the motor. For example, if the motor hears a PWM message when it is set to accept them, it will only listen to PWM messages until it is reset. 
-  If left in Autodetect mode, the motor should listen for all of the supported Hobby protocols to try to determine which one you are using. The motor can sometimes have trouble automatically discovering the protocol though 
-  depending on the flight controller. In those cases, it is best to set the motor to only listen for a specific type of protocol. For this reason, we will explicitly set the protocol to listen to in this setup. The proper value
-  for this configuration depends on if you are using PWM or DSHOT
-  
-  * **PWM Setup**: Set the Communication protocol to **Standard PWM** to listen to PWM messages in version 1.2.6 and later of the Control Center. In Version 1.2.5 and earlier, you will need to select OneShot125
-    due to a Control Center bug. See the bug warning below for details.
-  * **DSHOT Setup**: Set this to **DSHOT600** in version 1.2.6 and later of the Control Center. In Version 1.2.5 and earlier, you will need to select DSHOT1200 due to a Control Center bug. See the bug warning below for details.
+FC 2D/3D Mode
+##############
+This parameter tells the module if the flight controller wants it to operate in 2D mode (spinning in one direction) or 3D mode (able to spin in both directions). For more detail on
+what exactly these modes and how to configure the module to spin in both directions when receiving throttle commands, see the :ref:`manual_throttle` section of the Feature Reference Manual.
```

Comment:
For more details on these modes and how to configure the module's spinning behaviors, ...